db/migrate/20200922063521_add_default_to_user_column_company_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToUserColumnCompanyId < ActiveRecord::Migration[6.0]
+  def change
+    change_column_default :users, :company_id, 1
```

Comment:
why?

Reply:
multitenancy has not been merged yet. Once it is merged, I will get this from subdomain.

Reply:
remove this migration, we don't need this